src/System.DirectoryServices.AccountManagement/src/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard;
+      uap-Windows_NT;
```

Comment:
Why do we need the uap config? Isn't it the same as the netstdandard non-windows config?

Reply:
No, uap would be mapped as the netstandard-windows configuration. I just validated locally and if I remove the uap configuration then when building the src for uap we do get a full implementation assembly for netstandard instead of a PNS assembly. 

That is why I added this uap configurations to create the PNS assembly